ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -370,9 +418,16 @@ export const WorkspaceEdit = fp.flow(withCurrentWorkspace(), withCdrVersions(),
       this.setState({billingAccounts});
     }
 
+    async requestBillingScopeThenFetchBillingAccount() {
+      this.setState({fetchBillingAccountLoading: true});
```

Comment:
I think it would be good to also initialize this to true when we render the component. When I first visit the edit page, there is just a blank dropdown while the billing account loads in (if I _have_ billing scope, at least. Other scenario untested.

Reply:
I also use the state in initialBillingAccountLoad now. It seems to be better.